server/service/integration_enterprise_test.go

```diff
@@ -4604,11 +4605,12 @@ func (s *integrationEnterpriseTestSuite) TestRunHostSavedScript() {
 	err = s.ds.MarkHostsSeen(ctx, []uint{host.ID}, time.Now())
 	require.NoError(t, err)
 
-	// attempt to create a valid sync script execution request, fails because the
-	// host has a pending script execution
-	res = s.Do("POST", "/api/latest/fleet/scripts/run/sync", fleet.HostScriptRequestPayload{HostID: host.ID, ScriptID: &savedNoTmScript.ID}, http.StatusConflict)
-	errMsg = extractServerErrorText(res.Body)
-	require.Contains(t, errMsg, fleet.RunScriptAlreadyRunningErrMsg)
+	// TODO(mna): adjust after changes to API validations (i.e. /sync will fail if anything is in the queue, while async would enqueue without error)
```

Comment:
I.e. to address after the ` Run script endpoint changes` and `Run live script endpoint changes` bullets of the ticket are implemented.